pyxrf/model/draw_image_rgb.py

```diff
@@ -0,0 +1,353 @@
+# COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,           #
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES   #
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR   #
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)   #
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,  #
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OTHERWISE) ARISING   #
+# IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE   #
+# POSSIBILITY OF SUCH DAMAGE.                                          #
+########################################################################
+from __future__ import absolute_import
+__author__ = 'Li Li'
+
+import six
+import numpy as np
+from collections import OrderedDict
+from matplotlib.figure import Figure
+import matplotlib.pyplot as plt
+from matplotlib.colors import LogNorm
+#import matplotlib.cm as cm
+#from mpl_toolkits.axes_grid1 import ImageGrid
+import matplotlib.patches as mpatches
+from mpl_toolkits.axes_grid1.axes_rgb import make_rgb_axes, RGBAxes
+from atom.api import Atom, Str, observe, Typed, Int, List, Dict, Bool
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class DrawImageRGB(Atom):
+    """
+    This class draws RGB image.
+
+    Attributes
+    ----------
+    fig : object
+        matplotlib Figure
+    file_name : str
+    stat_dict : dict
+        determine which image to show
+    data_dict : dict
+        multiple data sets to plot, such as fit data, or roi data
+    data_dict_keys : list
+    data_opt : int
+        index to show which data is chosen to plot
+    dict_to_plot : dict
+        selected data dict to plot, i.e., fitting data or roi is selected
+    items_in_selected_group : list
+        keys of dict_to_plot
+    scale_opt : str
+        linear or log plot
+    color_opt : str
+        orange or gray plot
+    scaler_norm_dict : dict
+        scaler normalization data, from data_dict
+    scaler_items : list
+        keys of scaler_norm_dict
+    scaler_name_index : int
+        index to select on GUI level
+    scaler_data : None or numpy
+        selected scaler data
+    plot_all : Bool
+        to control plot all of the data or not
+    """
+
+    fig = Typed(Figure)
+    stat_dict = Dict()
+    data_dict = Dict()
+    data_dict_keys = List()
+    data_opt = Int(0)
+    #plot_opt = Int(0)
+    #plot_item = Str()
+    dict_to_plot = Dict()
+    items_in_selected_group = List()
+    scale_opt = Str('Linear')
+    scaler_norm_dict = Dict()
+    scaler_items = List()
+    scaler_name_index = Int()
+    scaler_data = Typed(object)
+    plot_all = Bool(False)
+
+    rgb_name_list = List()
+    index_red = Int(0)
+    index_green = Int(1)
+    index_blue = Int(2)
+
+    def __init__(self):
+        self.fig = plt.figure(figsize=(6,6))
+        self.rgb_name_list = ['R', 'G', 'B']
+
+    def data_dict_update(self, change):
```

Comment:
i know this is fussy, but this parameter named `change` does not quite match your docstring where this parameter is called `changed`